lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -422,24 +422,20 @@ void CommandInterpreter::Initialize() {
   cmd_obj_sp = GetCommandSPExact("process launch");
   if (cmd_obj_sp) {
     alias_arguments_vector_sp = std::make_shared<OptionArgVector>();
-#if defined(__arm__) || defined(__arm64__) || defined(__aarch64__)
+#if defined(__APPLE__)
+#if defined(TARGET_OS_IPHONE)
     AddAlias("r", cmd_obj_sp, "--");
     AddAlias("run", cmd_obj_sp, "--");
 #else
-#if defined(__APPLE__)
```

Comment:
Won't removing the #if here change how this works on other operating systems (e.g: Linux)?

Reply:
Yep, but I believe that the current behavior was not the intention when the #if's were added originally. I didn't get any complaints after I landed this upstream, so I'm going to assume that was indeed the case.